styles/styles.css

```diff
@@ -614,3 +614,79 @@ main .divider{
   margin: var( --spacer-layout-05) 0;
   border: 1px solid var(--neutral-sand);
   }
+
+
+main .section.leadership-head {
+  padding: 0 32px;
+}
+
+main .section.leadership-head h1 {
+  margin: 5px 0;
+}
+
+main .section.leadership-head p {
+  margin: 5px 0;
+}
+
+main .section.leadership-head .columns-2-cols>div>div:last-child {
+  margin: 0 -45px;
+}
+
+.leadership-head img {
+  clip-path: circle(50% at 50% 100%);
+  margin-bottom: -3px;
+}
+
+main .section.leadership-content {
+  background: #fff;
+  padding: 39px 0;
+}
+
+.leadership-content img {
+  border-radius: 15px;
+}
+
+main .section.leadership-content .columns>div>div {
+  margin: 25px 0;
+}
+main .section.leadership-content .columns>div>div>p {
+  font-style: normal;
+  font-size: 28px;
+  line-height: 120%;
+  letter-spacing: 0.0175em;
+}
+
+main .section.leadership-content .columns>div>div>h6 {
+  font-family: 'Alliance No.1';
```

Comment:
Everything else looks good. Just the fonts are weird. 
you can update this line to use the global variable for this font like below:

    font-family: var(--sans-serif-font-regular);
    
    Look at lazy-styles.css for the font definitions and styles.css lines 47-52 for variables pointing to the fonts you need.

Reply:
got it, will update it. I just copied the font name from figma

Reply:
@amol-anand done